src/components/image/image.tsx

```diff
@@ -39,13 +41,12 @@ export default function Image(props: ImageProps) {
         onLoad={() => {
           setIsLoading(false)
         }}
+        className={!props.width && !props.height ? 'rustic-full-width' : ''}
```

Comment:
The correct global attribute for html tags is `class` not `className`. 
Refer https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#try_it

Reply:
React components require className to be used I believe since class is a reserved keyword. I get this error when using class:
<img width="648" alt="Screenshot 2024-03-06 at 3 51 34 PM" src="https://github.com/rustic-ai/ui-components/assets/111031789/e23dd664-51b2-4f54-878b-a7815082d670">

Ref: https://react.dev/learn#adding-styles